packages/core/src/devtools/store.test.ts

```diff
@@ -0,0 +1,66 @@
+import { SEQUENCE_PREFIX } from '../constants';
+import { makeStyles } from '../makeStyles';
+import { mergeClasses } from '../mergeClasses';
+import { createDOMRenderer } from '../renderer/createDOMRenderer';
+import { MakeStylesOptions } from '../types';
+import { MK_DEBUG } from './store';
+
+jest.mock('./isDevToolsEnabled.ts', () => ({
```

Comment:
```suggestion
jest.mock('./isDevToolsEnabled', () => ({
```

nit: It should work even without an extension